src/mudlet.pro

```diff
@@ -1189,7 +1189,7 @@ macx {
 
         !exists($$SPARKLE_PATH) {
             message("Sparkle CocoaPod is missing, running 'pod install' to get it...")
```

Comment:
How about updating the message to match the QMake one - change:
```C++
            message("Sparkle CocoaPod is missing, running 'pod install' to get it...")
```
to:
```C++
            message("Sparkle CocoaPod is missing, updating pod repo && installing it...")
```

Also,  I am not sure that the existing code will always work to detect an out of date Sparkle CocoaPod on a developer's system where the source/build filesystem tree is not created anew for each new version...